.travis.yml

```diff
@@ -0,0 +1,5 @@
+script:
+- docker build --target test --tag my-test-image .
+- docker run my-test-image ./todo_app/tests/todo_view_model_test.py
+- docker run my-test-image ./todo_app/tests/test_client.py
+- docker run -e API_KEY -e API_TOKEN -e BOARD_NAME -e MEMBERS_NAME -e TO_DO_LIST_ID -e DOING_LIST_ID -e DONE_LIST_ID  -e FLASK_APP -e FLASK_ENV -e SECRET_KEY my-test-image  todo_app/tests_e2e
```

Comment:
The e2e tests shouldn't need BOARD_NAME or the list IDs because it creates a temporary board.

MEMBERS_NAME is not used by your app

Things like the FLASK_APP and FLASK_ENV could be hardcoded in the Dockerfile with `ENV` lines